packages/mdc-select/README.md

```diff
@@ -291,6 +291,35 @@ programmatically select a disabled list item in the enhanced select.
 </div>
 ```
 
+#### Enhanced Select Accessibility (a11y)
+
+In order to have an accessible component for users, it's recommended that you follow the WAI-ARIA example for
+[Collapsible Dropdown Listbox](https://www.w3.org/TR/wai-aria-practices/examples/listbox/listbox-collapsible.html). An
+example of the enhanced select component with all the necessary aria attributes is listed below.
+
+```html
+<div class="mdc-select">
+  <i class="mdc-select__dropdown-icon"></i>
+  <div id="demo-selected-text" class="mdc-select__selected-text" role="button" aria-haspopup="listbox" aria-labelledby="demo-label demo-selected-text">Vegetables</div>
```

Comment:
Does this need `aria-controls`?

> If the element with the aria-expanded attribute controls the expansion of another grouping container that is not 'owned by' the element, the author SHOULD reference the container by using the aria-controls attribute.

Reply:
I'm using `aria-controls` for the helper text and the way it was written for text field it won't support us using `aria-controls` for anything other than the helper text. Maybe switch the helper text logic to use `data-has-helper-text` to instantiate the MDCSelectHelperText and add multiple `aria-controls`.

The example I worked from on the WAI-ARIA site didn't use `aria-controls` either so I wasn't sure it was required.

Reply:
Huh, OK. Well, it's a "SHOULD" so that's more like strongly encouraged than outright required.